_posts/2017-11-11-environments.md

```diff
@@ -1,62 +1,75 @@
 ---
 layout: post
-title: Working with different package versions
+title: Managing software versioning using Conda environments
```

Comment:
I wouldn't stick to just conda here. For some people that would mean switching their complete python stack (and most scientists I know are averse to switching working systems). The idea behind the post is to show people how they can use newer version of MDAnalysis while still keeping their old scripts running.